src/test/scala/com/itechart/ny_accidents/utils/PostgisUtilsTest.scala

```diff
@@ -0,0 +1,17 @@
+package com.itechart.ny_accidents.utils
+
+import com.vividsolutions.jts.geom.{Coordinate, GeometryFactory}
+import org.scalatest.FunSpec
+
+class PostgisUtilsTest extends FunSpec {
+  describe("A PostgisUtils") {
+    it("should create postgis Point in correct format") {
```

Comment:
Test description should explain what exactly is "correct"

Reply:
btw, which SRID are you using to create polygons?

Reply:
If i'm not mistaken in datasets SRID = 0, so, we use it now. But later we will convert it into NY SRID

Reply:
https://app.vivifyscrum.com/boards/61175/NEAF-71